README.md

```diff
@@ -1 +1,60 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This project contains index.html file with a header and style.css file to style that header.
+> The project demonestrat the idea of adding file to .gitignore which make the files stay locally and  not push to github reposotary.
+
+
+## Built With
+
+- HTML&CSS
+- VScode 
+
+## Live Demo (if available)
+
+![Capture](https://user-images.githubusercontent.com/5276238/153204533-cad1759d-36a4-422a-8017-d8f15bb0a1a8.PNG)
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+VScode
+
+
+## Authors
+
+👤 **Maen Al-khraisha**
+
+- GitHub: [@githubhandle](https://github.com/maen1980)
+- Twitter: [@twitterhandle](https://twitter.com/AlkhryshaM)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ma-en-mohammad-303930100/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
This MIT license link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

Reply:
thank you for your help, comment is done.
take a look and give what you think. :)